### DISCUSSION TITLE
How to set validation message from the backend in form?

### DISCUSSION THREAD

**Discussion by fahmiegerton** [vueComponent/ant-design-vue#5077] [Q&A]
I need to set error message retrieved from the backend (laravel) and set it on every field if there is an error. The structure may look like this:
```
{
  'errors': {
     'email' : ['The Email field is required'],
     'password' : ['The Password field is required'] 
  }
}
```

In vuetify, I just set in the prop error-messages in one of the form input component (`error-messages=data.errors.email` then it picked up automatically the first one). The question is, how to do like this in ant-design-vue 3 ? Couldn't find any. I tried [this](https://stackoverflow.com/questions/60024729/vue-ant-design-display-client-side-as-well-as-server-side-validations-in-ants) but I don't want to use `createForm` since I already use `form` from inertia. Also tried [this](https://stanleytsau.me/ant-design-vue-form-display-error-message-from-backend/) too but I barely to understand that. Now I'm stuck, I'm hoping someone could help me, thanks.

**Comment by xiangnanscu** [comment]
Can't imagine antd vue is so weak to provide such a basic feature. write an Login form need this feature apparently.
